pkg/cmd/server/origin/master.go

```diff
@@ -132,23 +133,15 @@ import (
 )
 
 const (
-	LegacyOpenShiftAPIPrefix = "/osapi" // TODO: make configurable
-	OpenShiftAPIPrefix       = "/oapi"  // TODO: make configurable
-	OpenShiftAPIV1           = "v1"
-	OpenShiftAPIPrefixV1     = OpenShiftAPIPrefix + "/" + OpenShiftAPIV1
-	swaggerAPIPrefix         = "/swaggerapi/"
-	openAPIServePath         = "/swagger.json"
+	swaggerAPIPrefix = "/swaggerapi/"
+	openAPIServePath = "/swagger.json"
```

Comment:
It's a shame this isn't a constant/config option somewhere upstream like swagger api prefix.